docs/install/index.md

```diff
@@ -1,25 +1,122 @@
 ---
 title: Install ORT
-parent: How to
+has_children: false
 nav_order: 1
 ---
 
-# Install ONNX Runtime
+# Install ONNX Runtime (ORT)
 {: .no_toc }
 
 See the [installation matrix](https://onnxruntime.ai) for recommended instructions for desired combinations of target operating system, hardware, accelerator, and language. 
 
-Details on OS versions, compilers, language versions, dependent libraries, etc can be found under [Compatibility](../resources/compatibility.md#Environment-compatibility).
+Details on OS versions, compilers, language versions, dependent libraries, etc can be found under [Compatibility](../reference/compatibility).
+
 
 ## Contents
 {: .no_toc }
 
 * TOC placeholder
 {:toc}
 
-## Inference
+## Python Installs
+
+### Install ONNX Runtime (ORT)
+{: .no_toc }
+
+```bash
+pip install onnxruntime
+```
+```bash
+pip install onnxruntime-gpu
```

Comment:
You only need one of these, not both. And i think there is some work to actually make it into a single package. Check with Faith.

Reply:
I knew this but I added both for quick reference purposes. My thought is people will come here to see what/how to install based on their project. Maybe I should add some more explanation around when to install what on the next iteration? I will ask Faith about when that update is coming to combine it to one.